docs/learning/ui/appui-react/IModelViewportControl.md

```diff
@@ -0,0 +1,27 @@
+# IModelViewportControl
+
+The [IModelViewportControl]($appui-react) is a component based on the [ViewportContentControl]($imodel-components-react) that includes some convenience features for the iTwinjs app developer.
+
+### IModelViewportControlOptions
+
+In addition to connecting to the iModelConnection property from the AppUi Redux store, IModelViewportControl offers configuration via a set of optional props in the interface [IModelViewportControlOptions]($appui-react):
+
+* viewState: Can point to a [ViewStateProp]($core-frontend) or a function that returns a ViewStateProp.
+* iModelConnection: A connection to an [IModelDb]($backend) hosted on the backend, or a function that returns the [IModelConnection]($core-frontend).
+* featureOptions: A map of options to set the state of optional features in the view. For example, to enable automatic display of a timeline component for models containing a schedule script. This is the property you can use to turn on the animation timelines in the [DefaultViewOverlay]($appui-react).
+* bgColor: If there is no ViewState or IModelConnection defined for the Viewport, defining this optional prop allows you to set the background color.
```

Comment:
I think we should make this more clear that bgColor is only used if a ScreenViewport cannot be created using the ViewState and IModelConnection.